tests/test_icml_conference.py

```diff
@@ -3830,6 +3831,113 @@ def test_comment_stage(self, openreview_client, helpers):
+                    'value': {
+                        'param': {
+                            'type': 'string',
+                            'maxLength': 500,
+                            'optional': True,
+                            'deletable': True
+                        }
+                    }
+                },
+                'comment': {
+                    'order': 2,
+                    'description': 'Your comment or reply (max 5000 characters). Add formatting using Markdown and formulas using LaTeX. For more information see https://openreview.net/faq',
+                    'value': {
+                        'param': {
+                            'type': 'string',
+                            'maxLength': 5000,
+                            'markdown': True,
+                            'input': 'textarea'
+                        }
+                    }
+                }
+            },
+            multi_reply=True
+        )
+        venue.create_custom_stage()
+        helpers.await_queue_edit(openreview_client, 'ICML.cc/2023/Conference/-/Author_AC_Confidential_Comment-0-1', count=1)
+
+        assert len(openreview_client.get_invitations(invitation='ICML.cc/2023/Conference/-/Author_AC_Confidential_Comment')) == 100
+        invitation = openreview_client.get_invitation('ICML.cc/2023/Conference/Submission1/-/Author_AC_Confidential_Comment')
+        assert invitation.invitees == [
+            'ICML.cc/2023/Conference/Program_Chairs',
+            'ICML.cc/2023/Conference/Submission1/Area_Chairs',
+            'ICML.cc/2023/Conference/Submission1/Authors'
+        ]
+
+        author_client = openreview.api.OpenReviewClient(username='peter@mail.com', password=helpers.strong_password)
+        confidential_comment_edit = author_client.post_note_edit(
+            invitation='ICML.cc/2023/Conference/Submission1/-/Author_AC_Confidential_Comment',
+            signatures=['ICML.cc/2023/Conference/Submission1/Authors'],
+            note=openreview.api.Note(
+                replyto=submissions[0].id,
+                content={
+                    'comment': { 'value': 'Author confidential comment to AC' },
+                }
+            )
+        )
+        helpers.await_queue_edit(openreview_client, edit_id=confidential_comment_edit['id'])
+
+        confidential_comment = openreview_client.get_notes(invitation='ICML.cc/2023/Conference/Submission1/-/Author_AC_Confidential_Comment')[0]
+        assert confidential_comment.readers == [
+            'ICML.cc/2023/Conference/Program_Chairs',
+            'ICML.cc/2023/Conference/Submission1/Senior_Area_Chairs',
+            'ICML.cc/2023/Conference/Submission1/Area_Chairs',
+            'ICML.cc/2023/Conference/Submission1/Authors'
+        ]
+
+        # Check messages sent to readers
+        messages = openreview_client.get_messages(subject = '[ICML 2023] A author ac confidential comment has been received on your.*')
```

Comment:
Thanks for adding the test, I think we should add the logic to use "An" when the invitation name stats with a vowel.

Reply:
Here is an example from the Journal:

```
vowels = ['a', 'e', 'i', 'o', 'u']
before_invitation = 'An' if lower_formatted_invitation[0] in vowels else 'A'
```

Reply:
Fixed now